AppBuilder/platform/dataFields/ABFieldFile.js

```diff
@@ -48,8 +48,7 @@ module.exports = class ABFieldFile extends ABFieldFileCore {
                   // TODO: a reminder that you still got alot on the server to do!
                   webix.alert({
                      title: "!! TODO !!",
-                     text:
-                        "Tell a Developer to actually pay attention to this!",
+                     text: "Tell a Developer to actually pay attention to this!",
```

Comment:
I mean ... I'm thinking that at some point we actually need to pay attention to this.